io/tests/src/test/java/org/wildfly/extension/io/IOSubsystemTestCase.java

```diff
@@ -108,26 +104,5 @@ protected RunningMode getRunningMode() {
         };
     }
 
-    protected static final OptionAttributeDefinition ENABLED_PROTOCOLS = OptionAttributeDefinition.builder("enabled-protocols", Options.SSL_ENABLED_PROTOCOLS)
-            .setRequired(false)
-            .setFlags(AttributeAccess.Flag.RESTART_ALL_SERVICES)
-            .setAllowExpression(true)
-            .build();
-
-    @Test
-    public void testSequence() throws Exception {
```

Comment:
why did you remove that test?

Reply:
I think this must have been removed as a part of the original PR.  I could try adding it back.